sound/hda/intel-dsp-config.c

```diff
@@ -307,6 +307,10 @@ static const struct config_entry config_table[] = {
 		.flags = FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE,
 		.device = 0xa0c8,
 	},
+	{
+		.flags = FLAG_SOF,
```

Comment:
```suggestion
		.flags = FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE,
```

Reply:
@xiulipan It was my first version. But after talking with @mengdonglin , it seems that SST driver doesn't support TGL-H. In 
 other words, SOF driver is the only audio driver which supports TGL-H. That's why I changed the flag to FLAG_SOF.

Reply:
@bardliao @mengdonglin do we got SST support on TGL-U?

Reply:
@xiulipan We don't have SST support on TGL-U

Reply:
@bardliao @mengdonglin Then I am thinking we may also need a clean up for above few lines for TGL-U (or EHL)
We should only have `.flags = FLAG_SOF,` for them too, right?

Reply:
> @bardliao @mengdonglin Then I am thinking we may also need a clean up for above few lines for TGL-U (or EHL)
> We should only have `.flags = FLAG_SOF,` for them too, right?

Yes, I will do it.